picard/ui/infodialog.py

```diff
@@ -32,6 +32,33 @@
 from picard.ui.ui_infodialog import Ui_InfoDialog
 
 
+class ArtworkCoverWidget(QtWidgets.QWidget):
+    """A QWidget that can be added to artwork column cell of ArtworkTable."""
+
+    SIZE = 170
```

Comment:
Not sure if this should be caps.
Also maybe use a tuple here instead?

Reply:
I used caps because it is constant, and since height=width in this case, only one value is needed.
I don't feel the need to change it.